<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaDialog</class>
 <widget class="QDialog" name="MaDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>422</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Micro Array</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel,QSpinBox,QDoubleSpinBox,QPushButton,QToolButton,QLineEdit,QComboBox{
	min-height:48px;
	font: 12pt &quot;黑体&quot;;
}

QGroupBox{
	font: Bold 14pt &quot;黑体&quot;;
}


QRadioButton::indicator { 
	width: 32px;
	height: 32px;
}
/*单选框未选中样式*/
QRadioButton::indicator::unchecked {
	image: url(:/icons/uncheck.png);
}
/*单选框选中样式*/
QRadioButton::indicator::checked { 
	image: url(:/icons/check.png);
}
/*RadioButton和checkbox字体和间距设置*/
QRadioButton ,QCheckBox{
	spacing: 5px;
	font: 12pt &quot;黑体&quot;;
}

/*spinbox 抬起样式*/
QTimeEdit::up-button,QDoubleSpinBox::up-button,QSpinBox::up-button {subcontrol-origin:border;
    subcontrol-position:right;
	image: url(:/icons/up_left.png);
	background-color: rgb(255, 255, 255);
    width: 36px;
    height: 50px;       
}
QTimeEdit::down-button,QDoubleSpinBox::down-button,QSpinBox::down-button {subcontrol-origin:border;
    subcontrol-position:left;
	image: url(:/icons/up_right.png);
	background-color: rgb(255, 255, 255);
    width: 36px;
    height: 50px;
}

/*按钮按下样式*/
QTimeEdit::up-button:pressed,QDoubleSpinBox::up-button:pressed,QSpinBox::up-button:pressed{subcontrol-origin:border;
    subcontrol-position:right;
    image: url(:/icons/up_left.png);
	background-color: rgb(255, 255, 255);
    width: 35px;
    height: 49px;       
}
  
QTimeEdit::down-button:pressed,QDoubleSpinBox::down-button:pressed,QSpinBox::down-button:pressed,QSpinBox::down-button:pressed{
    subcontrol-position:left;
    image: url(:/icons/up_right.png);
	background-color: rgb(255, 255, 255);
    width: 35px;
    height: 49px;   
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="1,3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="measureGBox">
         <property name="title">
          <string>Measure</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QRadioButton" name="naRBtn">
            <property name="text">
             <string>Nucleic Acid</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="ptRBtn">
            <property name="text">
             <string>Protein</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="methodGBox">
         <property name="title">
          <string>Method</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
            <item>
             <widget class="QLabel" name="typeLabel">
              <property name="text">
               <string>Type</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="typeComboBox">
              <property name="iconSize">
               <size>
                <width>48</width>
                <height>48</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,1">
            <item>
             <widget class="QLabel" name="waveLabel">
              <property name="text">
               <string>Wave</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="waveSpinBox">
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="suffix">
               <string> nm</string>
              </property>
              <property name="minimum">
               <number>200</number>
              </property>
              <property name="maximum">
               <number>850</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
            <item>
             <widget class="QLabel" name="ceofLabel">
              <property name="text">
               <string>Coef</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="ceofDoubleSpinBox">
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1">
            <item>
             <widget class="QLabel" name="uintLabel">
              <property name="text">
               <string>Unit</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="uintComboBox">
              <property name="iconSize">
               <size>
                <width>48</width>
                <height>48</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="dyeGBox">
       <property name="title">
        <string>Dye</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QTableWidget" name="tableWidget">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>1/M*cm</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Wave Length</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>260nm%</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>280nm%</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="customGBox">
          <property name="title">
           <string>Custom</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="nameLabel">
             <property name="text">
              <string>Name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="nameLineEdit"/>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>1/M*cm</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBox">
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="maximum">
              <number>10000000</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="Label">
             <property name="text">
              <string>Wave Length</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="wlSpinBox">
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="suffix">
              <string> nm</string>
             </property>
             <property name="minimum">
              <number>200</number>
             </property>
             <property name="maximum">
              <number>850</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="addTBtn">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelPBtn">
       <property name="minimumSize">
        <size>
         <width>140</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="okPBtn">
       <property name="minimumSize">
        <size>
         <width>140</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
